chicago/models.py

```diff
@@ -172,6 +172,30 @@ def most_recent_past_city_council_meeting(cls):
         else:
             return None
 
+    @property
+    def clean_agenda_items(self):
```

Comment:
@fgregg I pulled this over from `django-councilmatic` and added a `prefetch_related` but I think I did it wrong. The page is very slow to load still